controllers/tasks.js

```diff
@@ -401,6 +401,27 @@ const assignTask = async (req, res) => {
   }
 };
 
+const updateOldTaskStatus = async (req, res) => {
+  try {
+    const allTasks = await tasks.fetchTasks();
+    const allOldTasks = allTasks.filter(
+      (task) => task.status.toUpperCase() === "COMPLETED" || task.status === "unassigned"
+    );
+    const updatedTasks = [];
+    for (const task of allOldTasks) {
+      if (task.status.toUpperCase() === "COMPLETED") {
```

Comment:
since COMPLETED is used at multiple places. can we make it into a constant or if it already exist can it be imported from there